fenix/app/build.gradle

```diff
@@ -460,7 +460,10 @@ nimbus {
 
 tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
     kotlinOptions {
-        freeCompilerArgs += "-opt-in=kotlinx.coroutines.ExperimentalCoroutinesApi"
+        freeCompilerArgs += [
+                "-opt-in=kotlinx.coroutines.ExperimentalCoroutinesApi",
+                "-opt-in=com.google.accompanist.pager.ExperimentalPagerApi",
```

Comment:
Let's opt for the manual `@OptIn` instead of adding a compiler args here. The reason being is that this accompanist library will soon be migrated into the Compose library and it would see easier to me to remove an one liner `@OptIn` than to revert the args here again.

Reply:
The [compose foundation pager](https://developer.android.com/jetpack/compose/layouts/pager) is also experimental, I think it would be easier to just update the flag here when we update to compose 1.4, as we will use the pager api.